src/Provider/Geonames.php

```diff
@@ -0,0 +1,269 @@
+<?php
```

Comment:
Yes, it's because of not using the latest master. I've moved the functionality to correct file.

I'd like to add just `getCountryInfo` in this PR. I'll add more methods in another PR.